src/lineage/snps/reader.py

```diff
@@ -78,6 +79,8 @@ def __call__(self):
                 return self.read_lineage_csv(file, comments)
             elif first_line.startswith("rsid"):
                 return self.read_generic_csv(file)
+            elif "vcf" in comments.lower():
```

Comment:
This was a tough one, I think it's common to look for a line staring with `"#CHROM"` to detect vcf, but there **should** be `"vcf"` in a header row per the [specs](https://samtools.github.io/hts-specs/VCFv4.2.pdf). This should work, but could be more explicit.

Reply:
Yeah it sounds like this should work.